pages/docs/concepts/application.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Application
+weight: 20
+---
+
+
+## What is an application?
+An application is any computer program or a group of them. 
+
+An application could also be a micro-service, IoT device (sensor), mainframe process, etc. Users may even write applications in different programming languages if they support one of the selected protocols. 
+
+## Why do we need applications?
+In Event-Driven Architecture (EDA), an application must be a producer, a consumer, or both. Applications must also use the protocols the server supports if they wish to connect and exchange messages.
+
+### Applications: producers and consumers
+```mermaid
+flowchart LR
+        A[PRODUCER application] --> B[message] 
+        B --> C[channel] 
+        C --> D[message] 
+        D --> F[CONSUMER application]
+```
+The diagram above describes the communication between a **producer application** and a **consumer application**. In this case, in your AsyncAPI file, you describe the `channel` via the [`Channel Bindings Object`](https://www.asyncapi.com/docs/reference/specification/v2.4.0#channelBindingsObject), which describes the protocol-specific definitions for `channel`s.
```

Comment:
```suggestion
The diagram above describes the communication between a **PRODUCER application** and a **CONSUMER application**.
```

I think names should match the same naming from the diagram

also I suggest remove the sentence about bindings. Or have it different below as separate paragraph

the things is you no necessarily need Channel Bindings, sometimes main Channel object is enough to describe the channel.

but yeah, I simply question if in application concept document we should now jump into a channel specific AsyncAPI details? shouldn't it go to `Channel concept` document?

maybe we should rather focus on explaining that in context of AsyncAPI documents, they are created from the application user perspective, and not the application perspective. That we describe what users can do with the app and not what app is doing.